.eslintrc.js

```diff
@@ -22,7 +22,10 @@ module.exports = {
         "plugin:@typescript-eslint/recommended",
         ...sharedRulePrettierExtends,
         "prettier/@typescript-eslint"
-      ]
+      ],
+      rules: {
+        "react/prop-types": "off"
+      }
```

Comment:
I think we do actually care about this, as this is a library that gets compiled to JavaScript, and we want anything that uses this library to complain about missing props, regardless of whether they're using TypeScript or not.